pkg/fixer/reporter.go

```diff
@@ -0,0 +1,31 @@
+package fixer
+
+import (
+	"fmt"
+	"io"
+)
+
+// Reporter writes a summary of the fixes applied to the output writer.
+func Reporter(outputWriter io.Writer, fixReport *Report) {
```

Comment:
I think It's fine to only support a single output format for the first implementation, but I think we should provide the `--format` flag from the start, with only a single option available, and this implementation here prepared for more formats getting added later.